org.gnunet.Messenger.json

```
@@ -0,0 +1,148 @@
+{
+  "app-id": "org.gnunet.Messenger",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "command": "messenger-gtk",
+  "finish-args": [
+    "--share=ipc",
+	"--socket=wayland",
+	"--socket=fallback-x11",
+	"--share=network",
+	"--socket=pulseaudio",
+	"--device=dri",
+
+	/* Home directory access to share files */
+	"--filesystem=home",
+	
+	/* Webcam access to scan QR codes */
+	"--device=all"
+  ],
```

```suggestion
  ],
  "cleanup": [
    "/bin/Magick*-config",
    "/include",
    "/lib/pkgconfig",
    "/share/aclocal",
    "/share/man",
    "/share/info",
    "*.la",
    "*.a"
  ],
```

also do you need all these installed binaries?

---

I assume not.